dts/arm/intel_socfpga_std/socfpga.dtsi

```diff
@@ -136,16 +136,16 @@
 			compatible = "snps,ethernet-cyclonev";
 			reg = <0xff700000 0x2000>;
 			interrupts = <0 115 4 IRQ_DEFAULT_PRIORITY>;
-			local-mac-address = [00 00 00 00 00 00];
+			emac-index = <0>;
 			status = "disabled";
 		};
 
 		gmac1: ethernet@ff702000 {
 			compatible = "snps,ethernet-cyclonev";
 			reg = <0xff702000 0x2000>;
 			interrupts = <0 120 4 IRQ_DEFAULT_PRIORITY>;
-			local-mac-address = [e6 88 2d 82 1f d4];
-			status = "okay";
+			emac-index = <1>;
+			status = "disabled";
```

Comment:
The commit message doesn't explain this change. Why are you removing `local-mac-address` and changing `status`?

Reply:
The "local-mac-address" property is moved from dts/arm/intel_socfpga_std/socfpga.dtsi to boards/arm/cyclonev_socdk/cyclonev_socdk.dts, since this value is dependent on the board and not universal to "intel_socfpga"

Reply:
Please add this explanation to the commit message